services/metadata/frontend/content/Tile.svelte

```diff
@@ -18,34 +18,30 @@
     border: 1px solid #000;
     border-radius: 5px;
     background-color: #fff;
-    padding: 10px 30px;
-    margin: 10px;
-    width: 240px;
-    height: 248px;
+    padding: 5px 5vw;
+    margin: 5px;
+    width: 75vw;
```

Comment:
I'm not sure about this: `width: 75vw;` makes it grow with the screen size, which can lead to very distorted tiles. Also, it looked better with the fixed height.

Reply:
Have you actually checked it? I made it for resolutions less than 768 only, so no, isn't going to happen like you're thinking. `vw` is relative to % of the viewport width, so 75 I've used will keep it inside the viewport.

Reply:
I have checked it. And I didn't mean it leaves the viewport, it just gets overly wide.

<img width="728" alt="image" src="https://user-images.githubusercontent.com/33053745/110127829-5f40a200-7dc6-11eb-8a37-1784f5dae63c.png">
(This is 700px width)

In my opinion, a card like this should definitely have a fixed x/y-ratio that is either square or slightly rectangular.

And I do think they should have a fixed height.

Reply:
This is a feature, not a bug. Join my room, I'll explain you why ;)